qt/scientific_interfaces/ISISReflectometry/GUI/MainWindow/MainWindowPresenter.cpp

```diff
@@ -5,12 +5,17 @@
 //     & Institut Laue - Langevin
 // SPDX - License - Identifier: GPL - 3.0 +
 #include "MainWindowPresenter.h"
+#include "../Common/Decoder.h"
```

Comment:
Could this path be `GUI/Common/Decoder.h` to be consistent with the other includes?